lute/templates/backup/index.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+
+{% block title %}Backups{% endblock %}
+
+{% block header %}Backups{% endblock %}
+
+{% block body %}
+
+{% if backups|length == 0 %}
+  <p>No backups have been made yet.</p>
+{% else %}
+
+<table class="table dataTable no-footer" style="width: 50%; margin: 0;">
+  <thead>
+    <tr>
+      <th>Backup File</th>
+      <th>Size</th>
+      <th>Last Modified</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for backup in backups %}
+    <tr>
+      <td>{{ backup.name }}</td>
+      <td>{{ backup.size }}</td>
+      <td>{{ backup.last_modified }}</td>
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
```

Comment:
We can style the table a bit differently -- remove some of the lastmodified milliseconds, ensure the size column doesn't wrap etc -- but that's for later.  This is super!

> <img width="75%" alt="image" src="https://github.com/jzohrab/lute-v3/assets/1637133/26c2bc90-0163-498d-9bfc-62f2de592419">

Reply:
Since we're touching this code, I can do this adjustment now. Also that's fairly easy to implement!